server/app/src/mocap_server.py

```diff
@@ -547,18 +547,13 @@ def __init__(self, bucket: str, deployment: str, singularity_image_path: str) ->
         self.index.addChangeListener(self.onChange)
         self.index.refreshIndex()
         self.index.registerPubSub()
-        # Subscribe to pings on the index
-        self.index.pubSub.subscribe(
-            "/PING/"+self.serverId, self.onPingReceived)
-        self.index.pubSub.subscribe("/PONG/#", self.onPongReceived)
 
-        cleanUpThread = threading.Thread(
-            target=self.cleanUpOtherDeadServersForever, daemon=True)
-        cleanUpThread.start()
+        # Subscribe to PubSub status checks.
+        self.index.pubSub.subscribe("/PUBSUB_STATUS/"+self.serverId, self.onPubSubStatusReceived)
```

Comment:
PubSub has a topic length character limit, which can be annoyingly short. I think it's like 70 characters or something crazy like that. If you attempt to send a longer topic length, I believe the server will just hang up on you (at least, the AWS server does). So it's worth checking that the topic is the right length. Perhaps we deliberately generate a shorter unique hash for the pubsub liveness checking topic?

Reply:
I could just truncate the server UUID to the first 16 characters, if that seems reasonable.